### ISSUE TITLE
Feature request: underline characters indicating indentation points

### ISSUE THREAD

**Issue by tomjaguarpaw** [haskell/haskell-mode#1584]
hi2-mode used to have little underline characters indicating which indentation points TAB would cycle though. Could we get that behaviour back in haskell-indentation-mode?

**Comment by tomjaguarpaw** [comment]
(Relevant context: https://www.reddit.com/r/haskell/comments/88ygjt/what_do_people_use_for_indentation_in_emacs/dwo4zpn/)

**Comment by purcell** [comment]
Yep, looks like this could be done quite easily in `haskell-indentation-indent-line` by creating a set of overlays each time it is called, and clearing them with a timer function.
